libsolutil/CommonData.cpp

```diff
@@ -180,14 +179,14 @@ bool solidity::util::isValidDecimal(string const& _string)
 	// No leading zeros
 	if (_string.front() == '0')
 		return false;
-	if (_string.find_first_not_of("0123456789") != string::npos)
+	if (_string.find_first_not_of("0123456789") != std::string::npos)
 		return false;
 	return true;
 }
 
-string solidity::util::formatAsStringOrNumber(string const& _value)
+std::string solidity::util::formatAsStringOrNumber(std::string const& _value)
 {
-	assertThrow(_value.length() <= 32, StringTooLong, "String to be formatted longer than 32 bytes.");
+	assertThrow(_value.length() <= 32, StringTooLong, "std::string to be formatted longer than 32 bytes.");
```

Comment:
Looks like your search and replace is not case-sensitive. It should be.

```suggestion
	assertThrow(_value.length() <= 32, StringTooLong, "String to be formatted longer than 32 bytes.");
```

Reply:
Yup, disabled case sensitivity when I was doing some regex checks in the file. I'll double check the PR again to see if there are more such occurrences.